lib/trento/infrastructure/software_updates/adapter/suma_http_executor.ex

```diff
@@ -200,6 +200,16 @@ defmodule Trento.Infrastructure.SoftwareUpdates.Suma.HttpExecutor do
     )
   end
 
+  def get_cert_der(ca_cert) do
+    ca_cert
+    |> :public_key.pem_decode()
+    |> Enum.map(&:public_key.pem_entry_decode/1)
+    |> Enum.map(&split_type_and_entry/1)
+    |> Enum.map(fn {ans1_type, ans1_entry} ->
+      :public_key.der_encode(ans1_type, ans1_entry)
```

Comment:
Typo: This should be `asn1`.

Reply:
here we go 70f2971b1b18370adf697ef23c3b916ceabc85ad